src/main/scala/is/hail/rvd/OrderedRVD.scala

```diff
@@ -48,17 +51,17 @@ class OrderedRVD private(
       partitioner,
       rdd.mapPartitionsWithIndex(f))
 
-  def mapPartitionsPreservesPartitioning(newTyp: OrderedRVDType)(f: (Iterator[RegionValue]) => Iterator[RegionValue]): OrderedRVD =
-    OrderedRVD(newTyp,
+  def mapPartitionsPreservesPartitioning(rowTyp: TStruct)
+    (f: (Iterator[RegionValue]) => Iterator[RegionValue])
+      : OrderedRVD =
```

Comment:
Is this a thing we're doing now, putting the colon on a line by itself?  I thought we liked "Patrick's" style:

```
  def (
    a1: T1,
    ...
    an: Tn
  ): RT
```

I'm not saying no, I'm just asking if we're changing/endorsing a new style

Reply:
just a formatting flub